web/src/store/modules/tracks.js

```diff
@@ -34,11 +35,17 @@ const actions = {
   },
   async fetchTrack({ commit }, payload) {
     const response = await client.get(
-      `/v1/reciters/${payload.reciter}/albums/${payload.album}/tracks/${payload.track}?include=lyrics`,
+      `/v1/reciters/${payload.reciter}/albums/${payload.album}/tracks/${payload.track}?include=reciter,album,lyrics`,
     );
     commit('FETCH_TRACK', {
       data: response.data,
     });
+    store.commit('reciters/FETCH_RECITER', {
```

Comment:
Instead of importing the `store` and doing this, you should do:
```js
commit('reciters/FETCH_RECITER', {
  data: response.data.reciter,
}, { root: true });
```